domain/src/main/kotlin/com/nohrd/bike/domain/BikeDataListener.kt

```diff
@@ -1,4 +1,4 @@
-package com.nohrd.bike.sdk
+package com.nohrd.bike.domain
```

Comment:
Don't share this between the `sdk` and `sdk-ftms` modules. The implementation for these two modules have very different reasons to change, even though the listeners behave the same right now. It is the client's responsibility to adapt these interfaces to their needs.